util/scripts/launch.sh

```diff
@@ -69,5 +67,11 @@ else
   exit 2
 fi
 
-# Launch the compiler.
-run_cli_tool_with_args "$@"
+gradlew="${base}/gradlew"
+
+# Launch the tool.
+if [ $# -eq 0 ]; then
+    "${gradlew}" -p "${base}" "org.lflang:cli:${tool}:run"
```

Comment:
:thinking: Does this mean that Gradle is needed as a dependency to be able to run `lfc`?

Reply:
Now, when I run `lfc` without arguments, instead of getting output directly from `lfc`, I get a message from `gradle`. I don't think this is what we want.

Reply:
I now see this is also what you were discussing above with @axmmisaka. What motivated this change to begin with?

Reply:
> Does this mean that Gradle is needed as a dependency to be able to run lfc?

Gradle is only needed in the developer setup, and since we use the wrapper it will install itself.

> Now, when I run lfc without arguments, instead of getting output directly from lfc, I get a message from gradle. I don't think this is what we want.

The output of lfc is printed directly. Gradle only adds a message to the bottom.

> I now see this is also what you were discussing above with @axmmisaka. What motivated this change to begin with?

To get rid of the launch shell scripts that search for a compiled jar and then invoke it and the build script. I think we should just embrace that we use gradle and not add additional logic around it. There are a few other reasons to, which I will add to my summary.